music_assistant/providers/opensubsonic/sonic_provider.py

```diff
@@ -457,11 +465,14 @@ async def get_artist_albums(self, prov_artist_id: str) -> list[Album]:
         if prov_artist_id == UNKNOWN_ARTIST_ID or prov_artist_id.startswith(NAVI_VARIOUS_PREFIX):
             return []
 
-        try:
-            sonic_artist: SonicArtist = await self._run_async(self.conn.get_artist, prov_artist_id)
-        except (ParameterError, DataNotFoundError) as e:
-            msg = f"Album {prov_artist_id} not found"
-            raise MediaNotFoundError(msg) from e
+        sonic_artist: Artist = await self.get_artist(prov_artist_id=prov_artist_id)
+        # try:
```

Comment:
Why is this code commented out? If it is incorrect, remove it. But your commit message does not explain why this is removed.